### PR TITLE
fix(website): Remove unnecessary "please" from examples, clarify error text copy guidance

### PR DESCRIPTION
**Pull request by serifluous**
- Remove unnecessary "please" from examples based on @ctecks 's guidance here https://www.figma.com/file/6yC4lctC4c44byRSG1DhVw?node-id=2:8458#98657815
- Clarify error text and help text relationship on form field pages.

### CODE REVIEW

File: packages/paste-website/src/pages/components/checkbox/index.mdx [hunk 1]
```diff
@@ -92,7 +92,7 @@ Checkbox groups are used when you have multiple binary choices to make but all c
 
 ### About Checkbox
 
-Use a checkbox to present a user with a binary (e.g. on/off) choice that is part of a form. A checkbox should not be used to apply a user's choice instantly. In that case, such as enabling a system feature, you should consider using the toggle switch component (being built in a coming sprint).
+Use a checkbox to present a user with a binary (e.g. on/off) choice that is part of a form. A checkbox should not be used to apply a user's choice instantly. In that case, such as enabling a system feature, you should consider using the toggle switch component ([coming soon(https://paste.twilio.design/roadmap/)).
```
Comment (richbachman) [inline packages/paste-website/src/pages/components/checkbox/index.mdx:95]:
```suggestion
Use a checkbox to present a user with a binary (e.g. on/off) choice that is part of a form. A checkbox should not be used to apply a user's choice instantly. In that case, such as enabling a system feature, you should consider using the toggle switch component ([coming soon](https://paste.twilio.design/roadmap/)).
```

**Review by richbachman — APPROVED** [review]
Just saw the link syntax error. Other than that, lgtm.

File: packages/paste-website/src/pages/components/checkbox/index.mdx (hunk 1 above)
Comment (serifluous) [reply packages/paste-website/src/pages/components/checkbox/index.mdx:95]: ahh! thanks for the catch.

**Review by zahnster — DISMISSED** [review]
Looks great overall, great content edits. I'm noticing some instances on the proposed Radio Group / Checkbox / Select / Textarea pages where some headlines are still in title case, while I see many of them have been changed out of title case. Would be great to have one more pass on this to ensure consistency throughout the pages.

I also see a headline on the Textarea page: "When to use an textarea" where an => a

File: packages/paste-website/src/pages/components/checkbox/index.mdx [hunk 2]
```diff
@@ -325,13 +321,13 @@ Error text should:
...
 
-There is no maximum number of checkboxes you can have in a group, but when it starts to get over 6 you might consider another component. Use your discretion to decide what is best for your use case. Some alternatives to look at when the number of options starts getting above 6 might be a Select or Combobox.
+There is no maximum number of checkboxes you can have in a group, but consider using another component when you have more than 6 options. Some alternatives might be a [Select](https://paste.twilio.design/components/select/) or [Combobox](https://paste.twilio.design/components/combobox/).
 
 <DoDont>
   <Do title="Do" body="Use checkboxes to enable multiple choice." />
   <Dont
     title="Don't"
-    body="Don't use checkboxes when you need to restrict selection to a single option, use a radio group or select instead."
+    body="Don't use checkboxes when you need to restrict selection to a single option. Use a Radio Group or Select instead."
```
Comment (andipants12) [inline packages/paste-website/src/pages/components/checkbox/index.mdx:330]: ❓ Should these be capitalized?

**Review by andipants12 — APPROVED** [review]

**Comment by TheSisb** [comment]
TODO: go into each of the pages that have been edited and make sure all headings are sentence case:
- Checkbox
- Combobox
- Date Picker
- Help Text
- Input
- Label
- Radio Group
- Select
- Textarea
- Time Picker

**Review by SiTaggart — APPROVED** [review]

**Pull request merged**